app/core/ViewLoadTimer.coffee

```diff
@@ -87,16 +91,57 @@ class ViewLoadTimer
 
       if @firstLoad
         entries = performance.getEntriesByType('resource').filter((r) => _.string.startsWith(r.name, location.origin))
+        essentialEntries = _.filter(entries, (entry) =>
+          return false if /vimeo/.test(entry.name)
+          return false if /TestView/.test(entry.name)
+          return false if /file\/db\//.test(entry.name) # Don't count DB images
+          return false if /\/db\/analytics\.log\.event/.test(entry.name) # Don't count analytics events
+          return true if /\/db\//.test(entry.name) # Do count DB data
+          return true if /chunks\/.*bundle.js/.test(entry.name)
+          return true if /app.js/.test(entry.name)
+          return true if /boot.js/.test(entry.name)
+          return true if /ace/.test(entry.name)
+          return true if /esper/.test(entry.name)
+          return true if /aether/.test(entry.name)
+          return true if /esper.modern.js/.test(entry.name)
+          return true if /lodash.js/.test(entry.name)
+          return true if /run-tests.js/.test(entry.name)
+          return true if /setImmediate.js/.test(entry.name)
+          return true if /web-dev-listener.js/.test(entry.name)
+          return true if /world/.test(entry.name)
+          return true if /app.css/.test(entry.name)
+          return false
```

Comment:
We need a strategy to keep this up to date. Is editing this part of further performance work?

Reply:
Resolved: Added a comment to webpack config to update this whitelist if we add entrypoints or static assets